mne/io/edf/edf.py

```diff
@@ -615,9 +615,12 @@ def _read_edf_header(fname, exclude):
                 continue
             if unit == 'uV':
                 edf_info['units'].append(1e-6)
+            if unit == 'V':
+                edf_info['units'].append(1)
```

Comment:
Move these two lines to the top such that the order is `V`, `mV`, and `uV`. You need to create `if`/`elif`/`else` branches in order for this to work (previously you used another `if` clause).

Reply:
I hope this would fix that issue
![image](https://user-images.githubusercontent.com/39642083/112205851-41cf5d00-8c3b-11eb-9b83-ed1737baff05.png)

Reply:
Yes, this looks good! You _could_ start with an `if` after the `continue` statement, but an `elif` is also fine!